libkrun/libkruntime.c

```diff
@@ -1,5 +1,10 @@
 /*
- * C support functions.
+ * C support functions for timing.
+ *
+ * Although this is a library, any error will exit indicating failure. Krun
```

Comment:
It is impolite to exit() in a library. Usually you would return an error condition and allow the consumer to decide what to do.

I can put words to this effect if that makes sense to you.